automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/apps/AppCreationSpec.scala

```diff
@@ -91,13 +93,18 @@ class AppCreationSpec extends GPAllocFixtureSpec with LeonardoTestUtils with GPA
             loggerIO.warn(
               s"AppCreationSpec: app ${googleProject.value}/${appName.value} did not finish deleting after 20 minutes. Result: $monitorDeleteResult"
             )
+            IO(Sam.user.deleteResource("kubernetes-app", appName.value)(ronCreds.makeAuthToken()))
```

Comment:
This doesn't seem correct -- maybe should be removed? Leo is supposed to delete the Sam resource when deletion finishes; also I think the Sam resource has a different name than the app name (a UUID).

Reply:
I feel like we added this because of some clean up we were having issues with. It may have been resolved by a time out extension we added. Will try a test run without these to verify that these aren't needed anymore.

I think the original intention was because we are now having to clean up a workspace, we may have issues in the instance where the workspace still has a child resource and cannot be cleaned up.